### ISSUE TITLE
Managed Collection checkboxes are missing labels

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6778]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), some (but not all) checkboxes on the `Managed/All Collections` tab are missing labels and therefore do not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/dashboard/collections?locale=en and click on `Managed Collections` or `All Collections` (if admin user) and run the Site Improve browser extension. This error does not appear on the `My Collections` tab

![maanged_collections_test](https://github.com/samvera/hyrax/assets/32644254/4acd6dfe-2ed2-4131-92da-47ae6b93ddcf)

**Comment by aahurford** [comment]
tested and confirmed in pg, but occurence text reads <input> (checkbox}

![Image](https://github.com/user-attachments/assets/068cb6b1-2208-4833-9d87-39f80585483b)

**Comment by aahurford** [comment]
looks good in dev and pg
